app/utils/json_data.py

```diff
@@ -0,0 +1,32 @@
+import requests
+import json
+from flask import url_for
+
+
+def fetch(job_id, url=None):
+
+    """Fetch the wanted JSON data
+
+    @param job_id: Id of a job.
+    @param url: URL from which to retrieve the data.
+    @return: the desired JSON data or None.
+    """
+
+
+    if url is None:
+        url = 'http://localhost:5000' + \
+              url_for('classifications_id', job_id=job_id)
+
+    try:
+        request = requests.get(url)
+
+        json_data = request.json()
+
+        data = json.dumps(dict(json_data.get('data')))
+        return data
+
+
+    except Exception as e:
+
+        print(repr(e))
```

Comment:
My fault, I forgot to review this file and I think I did it because of sum bug that I had in the development phase.
This was the best way I've found to retrieve the json date after the processing. But now I'm thinking that maybe would be better to retrieve the classification data directly from the worker, using redis. Do you agree? Otherwise another solution could be to store the port and the base URL in the configuration file and pass these parameter also to the init of  the server. In this way we will have all configuration parameters in the config file.